### PR TITLE
Fix basic zemu test

### PR DESCRIPTION
**Pull request by rllola**
`basic sign & verify` test not working for unclear reason. Got the transaction from `ledger-tendermint-rs`.

Now working. I had to modify the ledger app and update the way we process chunks. The JS app has also been updated in some ways.

### CODE REVIEW

File: js/src/index.js [hunk 1]
```diff
@@ -17,14 +17,16 @@
...
-import {errorCodeToString, P1_VALUES, processErrorResponse, serializePath} from "@zondax/ledger";
+import {errorCodeToString, P1_VALUES, processErrorResponse, serializePath,} from "@zondax/ledger";
 
 function processGetAddrResponse(response) {
   let partialResponse = response;
 
   const errorCodeData = partialResponse.slice(-2);
   const returnCode = errorCodeData[0] * 256 + errorCodeData[1];
 
+  console.log(PKLEN)
```
Comment (rllola) [inline js/src/index.js:28]: That should not be here...

File: tests_zemu/tests/test.js [hunk 2]
```diff
@@ -128,35 +100,51 @@ describe('Standard', function () {
...
+            console.log(pkResponse)
+
+            await Zemu.sleep(2000);
 
             // do not wait here..
-            const signatureRequest = app.sign(path, txBlob);
+            const signatureRequestInit = app.sign(path, txBlob);
             // Wait until we are not in the main menu
-            await sim.waitUntilScreenIsNot(sim.getMainMenuSnapshot());
+            //await sim.waitUntilScreenIsNot(sim.getMainMenuSnapshot());
```
Comment (rllola) [inline tests_zemu/tests/test.js:118]: Why is this commented ? Is this link to ledger UI glitch ?

**Comment by jleni** [comment]
Continuous integration is missing here. I will add it

**Comment by jleni** [comment]
Closing.. it is included in https://github.com/Zondax/ledger-tendermint/pull/3

**Pull request closed without merging**